tests/test_celery_unique.py

```diff
@@ -87,15 +86,15 @@ def test_does_not_delete_from_redis_when_not_found_in_redis(self):
         self.assertFalse(mock_redis_client_delete.called)
 
     def test_revokes_async_result_when_found_in_redis(self):
-        test_task_id = str(uuid4())
+        test_task_id = uuid4().hex.encode()
```

Comment:
That still returns a something that needs to be cast, otherwise it fails under py3

    AssertionError: b'9f058baf-ae2e-4f07-b5a6-f8fcc34a0ef8' != '9f058baf-ae2e-4f07-b5a6-f8fcc34a0ef8'